### PR TITLE
Di 26840 migrate to nodejs20 runtime incl aws sdk 3

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

File: lambda-handler/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {ExecuteAddonSync} from '@pepperi-addons/addon-infra-sdk/build/src/pepperi-lambda-handler/addons-index';
```
Comment (Chasky-H) [inline lambda-handler/index.ts:1]: what is this?

Comment (saargt) [reply lambda-handler/index.ts:1]: This is the call to the AIS Lambda handler code.
You can intevene and add some logic here, but PFS doesn't currently need that.

Comment (Chasky-H) [reply lambda-handler/index.ts:1]: this is a seperate project, right? Currently not in scope? 
If so, remove from this PR.

**Comment by Chasky-H** [comment]
Will all this work on lambdas with Node16?

**Pull request closed without merging**